tests/virt/conftest.py

```diff
@@ -5,95 +5,92 @@
+            if _is_psi_cluster:
+                failed_verifications_list.append("Cluster should be BM and not PSI")
 
     def _verify_cpumanager_workers(_schedulable_nodes):
-        LOGGER.info("Verify cluster nodes have CPU Manager labels")
-        if not any([node.labels.cpumanager == "true" for node in _schedulable_nodes]):
-            failed_verifications_list.append("Cluster does't have CPU Manager")
+        if any(item.get_closest_marker("cpu_manager") for item in request.session.items):
+            LOGGER.info("Verify cluster nodes have CPU Manager labels")
+            if not any([node.labels.cpumanager == "true" for node in _schedulable_nodes]):
+                failed_verifications_list.append("Cluster does't have CPU Manager")
 
     def _verify_gpu(_gpu_nodes, _nodes_with_supported_gpus):
-        LOGGER.info("Verify cluster nodes have enough supported GPU cards")
-        if not _gpu_nodes:
-            failed_verifications_list.append("Cluster doesn't have any GPU nodes")
-        if not _nodes_with_supported_gpus:
-            failed_verifications_list.append("Cluster doesn't have any nodes with supported GPUs")
-        if len(_nodes_with_supported_gpus) < 2:
-            failed_verifications_list.append("Cluster has only 1 node with GPU")
+        if any(item.get_closest_marker("gpu") for item in request.session.items):
+            LOGGER.info("Verify cluster nodes have enough supported GPU cards")
+            if not _gpu_nodes:
+                failed_verifications_list.append("Cluster doesn't have any GPU nodes")
+            if not _nodes_with_supported_gpus:
+                failed_verifications_list.append("Cluster doesn't have any nodes with supported GPUs")
+            if len(_nodes_with_supported_gpus) < 2:
+                failed_verifications_list.append(f"Cluster has only {len(_nodes_with_supported_gpus)} node with GPU")
 
     def _verfify_no_dpdk():
-        LOGGER.info("Verify cluster doesn't have DPDK enabled")
-        if PerformanceProfile(name="dpdk").exists:
-            failed_verifications_list.append("Cluster has DPDK enabled (DPDK is incomatible with NVIDIA GPU)")
-
-    def _verify_numa(_schedulable_nodes, _workers_utility_pods):
-        LOGGER.info("Verify cluster has NUMA")
-        cat_cmd = "cat /etc/kubernetes/kubelet.conf"
-        single_numa_node_cmd = f"{cat_cmd} | grep -i single-numa-node"
-        topology_manager_cmd = f"{cat_cmd} | grep -w TopologyManager"
-        for cmd in (single_numa_node_cmd, topology_manager_cmd):
-            for node in _schedulable_nodes:
-                pod_exec = ExecCommandOnPod(utility_pods=_workers_utility_pods, node=node)
-                out = pod_exec.exec(command=cmd, ignore_rc=True)
-                if not out:
-                    failed_verifications_list.append(f"Cluster does not have {cmd.split()[-1]} enabled")
+        if any(item.get_closest_marker("gpu") for item in request.session.items):
```

Comment:
you mean dpdk?

Reply:
virt doesn't have dpdk tests. but we need to check that if tests with gpu mark are  collected, then there should be no DPDK on cluster (as it's incompatible with vGPU)